### DISCUSSION TITLE
Sending emails from Supabase

### DISCUSSION THREAD

**Discussion by codepeach** [supabase/supabase#4729] [Questions]
I have a custom SMTP server using AWS' Simple Email Service and I was wondering if there is a way for me to integrate it with Supabase.

I am creating a social media platform and I want to send an email each time someone has commented on a particular user's post. Is there a way I can set a trigger such that each time there is an insert into the comments table, I can send a customized message through supabase?

**Comment by GaryAustin1** [comment]
You probably can either use function hooks, or a function using the http extension with a trigger on insert.
https://supabase.com/blog/2021/07/30/supabase-functions-updates
